.env.example

```diff
@@ -0,0 +1,21 @@
+PORT=8081
```

Comment:
We are not using this file anymore. We can delete it.

Reply:
Why? I think it is useful to have this file because if another dev needs to setup their dev environment could copy this file to .env.